s3cortxutils/s3confstore/ut/s3confstore_unit_tests.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+#!/usr/bin/python3.6
+
+from s3confstore.cortx_s3_confstore import S3CortxConfStore
+
+def test_uttest():
+  s3confstore = S3CortxConfStore()
+  s3confstore.uttest("ut_index")
```

Comment:
Pratyush - I guess this is not right way to right UTs... can you please refer s3recoverytool UT as suggested and write proper UT ?

Reply:
The current s3confstore.uttest() is the inbuilt UT written before, it exercises all the APIs. But ok, I'll also add the individual APIs separately in this UT.

Reply:
Point is not to invoke all APIs, but to test each entry-exit points of a sub-module, where the python test module would help, by mocking unwanted outside stuff, and just test the sub-module of our class.